Source/Core/DolphinQt/Config/Mapping/IOWindow.cpp

```diff
@@ -425,6 +437,23 @@ void IOWindow::ConnectWidgets()
   connect(m_expression_text, &QPlainTextEdit::textChanged,
           [this] { UpdateExpression(m_expression_text->toPlainText().toStdString()); });
 
+  connect(m_variables_combo, qOverload<int>(&QComboBox::activated), [this](int index) {
+    if (0 == index)
```

Comment:
```suggestion
    if (index == 0)
```

To be consistent with the line below. We also generally don't invert conditionals like this in the codebase.

Reply:
Actually I don't like the inverted condition check but that's how it's done in 2 places below that are doing the same exact thing. I was about to change them but then I left them and decided to keep consistency with them.

Reply:
Ok i've changed the ones below as well, looks better now.